packages/jellyfish-transaction/__tests__/script/dftx/dftx_loans/UpdateVault.test.ts

```diff
@@ -0,0 +1,89 @@
+import { SmartBuffer } from 'smart-buffer'
+import {
+  CUpdateVault,
+  UpdateVault
+} from '../../../../src/script/dftx/dftx_loans'
+import { OP_CODES } from '../../../../src/script'
+import { toBuffer, toOPCodes } from '../../../../src/script/_buffer'
+import { OP_DEFI_TX } from '../../../../src/script/dftx'
+
+it('should bi-directional buffer-object-buffer', () => {
+  const fixtures = [
```

Comment:
feels like same conditions.. are these included update - only ownerAddr and only scheme??

Reply:
Ya, I left it intentionally. 
This is what I think. For DFTXs, all the parameters must exists.

Reply:
And also, like the UTXOS tests in the RPC. I won't add them too.